ChangeLog

```diff
@@ -17,6 +17,13 @@
         * inst/NEWS.Rd: Idem
         * vignettes/rmd/Rcpp.bib: Idem
         * inst/bib/Rcpp.bib: Idem
+2020-07-01 Walter Somerville <waltersom@gmail.com>
```

Comment:
This should be 

- in chronological order
- with proper empty lines

Reply:
I just sent a trivial commit back to you for that so make sure you pull to your box.